src/tests/ftest/vmd/led.py

```diff
@@ -0,0 +1,136 @@
+        finally:
+            self.dmg.json.value = False
+
+        data = json.loads(result.stdout_text)
+        resp = data['response']
+        if data['error'] or len(resp['host_errors']) > 0:
+            if data['error']:
+                self.fail("dmg command failed: {}".format(data['error']))
+            else:
+                self.fail("dmg command failed: {}".format(resp['host_errors']))
+        uuid = []
+        for value in list(resp['host_storage_map'].values()):
+            if value['storage']['smd_info']['devices']:
+                for target in range(self.targets):
+                    uuid.append(value['storage']['smd_info']['devices'][target]['uuid'])
+        return uuid
+
+    def get_LED_status_value(self, device_id=None):
+        """Get LED Status value.
+
+        Args:
+            device_id (str): Device UUID
+        """
+        if device_id is None:
+            self.fail("No device id provided")
+
+        self.dmg.json.value = True
+        try:
+            result = self.dmg.storage_identify_device(uuid=device_id)
+        except process.CmdError as details:
+            self.fail("dmg command failed: {}".format(details))
+        finally:
+            self.dmg.json.value = False
+
+        data = json.loads(result.stdout_text)
+        resp = data['response']
+        if data['error'] or len(resp['host_errors']) > 0:
+            if data['error']:
+                self.fail("dmg command failed: {}".format(data['error']))
+            else:
+                self.fail("dmg command failed: {}".format(resp['host_errors']))
+        self.log.info(resp)
+
+    def set_device_faulty(self, device_id=None):
+        """Get a device to faulty state.
+
+        Args:
+            device_id (str): Device UUID
+        """
+        if device_id is None:
+            self.fail("No device id provided")
+
+        self.dmg.json.value = True
+        try:
+            result = self.dmg.storage_set_faulty(uuid=device_id)
+        except process.CmdError as details:
+            self.fail("dmg command failed: {}".format(details))
+        finally:
+            self.dmg.json.value = False
+
+        data = json.loads(result.stdout_text)
+        resp = data['response']
+        if data['error'] or len(resp['host_errors']) > 0:
+            if data['error']:
+                self.fail("dmg command failed: {}".format(data['error']))
+            else:
+                self.fail("dmg command failed: {}".format(resp['host_errors']))
+        self.log.info(resp)
+
+
+    def test_VMD_LED_Status(self):
+        """Jira ID: DAOS-10181
+
+        :avocado: tags=all,manual
+        :avocado: tags=hw,medium,ib2
+        :avocado: tags=vmd_led,faults
+        :avocado: tags=vmd_led_basic
+        """
+        dev_id = []
+        # Get the list of device ids.
+        dev_id = self.get_nvme_device_ids()
+        self.log.info("%s", dev_id)
+        for val in dev_id:
+            resp = self.get_LED_status_value(val)
```

Comment:
(pylint-assignment-from-no-return) Assigning result of a function call, where the function has no return